tests/testthat/helper.R

```diff
@@ -12,6 +12,6 @@ if (!nzchar(Sys.getenv("DEEPL_API_KEY"))) {
   Sys.setenv("DEEPL_API_KEY" = "lalala")
 }
 
-if (!nzchar(Sys.getenv("DEEPL_API_URL"))) {
-  Sys.setenv("DEEPL_API_URL" = "https://api.deepl.com")
-}
+# if (!nzchar(Sys.getenv("DEEPL_API_URL"))) {
```

Comment:
for info @xtimbeau, as it might influence your testing (I don't have access to a pro account at the moment)